src/richie/plugins/slider/factories.py

```diff
@@ -0,0 +1,41 @@
+"""
+Slider CMS plugin factories
+"""
+
+import factory
+import faker
+
+from .models import SlideItem, Slider
+
+
+class SliderFactory(factory.django.DjangoModelFactory):
+    """
+    Factory to create instance of a Slider.
+    """
+
+    title = factory.Faker("text", max_nb_chars=20)
+
+    class Meta:
+        model = Slider
+
+
+class SlideItemFactory(factory.django.DjangoModelFactory):
+    """
+    Factory to create instance of a SlideItem.
+    """
+
+    title = factory.Faker("text", max_nb_chars=20)
+    content = factory.Faker("text", max_nb_chars=42)
+    image = factory.SubFactory("richie.apps.core.factories.FilerImageFactory")
+    link_open_blank = factory.Faker("pybool")
+
+    class Meta:
+        model = SlideItem
+
+    @factory.lazy_attribute
+    def link_url(self):
+        """
+        Set a random url
+        """
+        instance = faker.Faker()
+        return instance.url()
```

Comment:
This doesn't work ?

```suggestion
    link_url = factory.Faker("url")

    class Meta:
        model = SlideItem
```

Reply:
It was an workaround for an old factoryboy bug i forgotten it was fixed long ago. Done.